index.html

```diff
@@ -160,5 +160,21 @@ <h3 class="pcol">LET’S CONNECT</h3>
                 </ul>
             </div>
            </div>
+           <section class="contact">
+            <div class="text-div">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing,
+                     a feature that you need built or a project that needs coding.
+                      I’d love to help with it </p>
+            </div>
+            <div class="form-div">
+<form class="form1" action="https://formspree.io/f/xpzewknr" method="post">
+    <input class="design" type="text" name="fullname" id="fullname" size="30" placeholder="Enter your Name here" required><br>
+    <input class="design" type="email" name="email" id="email" placeholder="Enter your Email" required><br>
+    <textarea name="message" id="message" cols="30" rows="10" size="500" placeholder="Enter your Message here" required></textarea>
```

Comment:
- [ ] As I mentioned before as a project requirement message field should only accept up to 500 chars. I have noticed that you used `size="500"` to do that but this is not working so I think it would be better if you use the `maxlength="500"` attribute for the requirement :+1: